src/tools/schema2kotlin.ts

```diff
@@ -393,9 +405,16 @@ ${classDef}${constructorArguments}${classInterface}
     ${this.opts.wasm ? `override var entityId = ""` : withFields(`init {
         ${this.fieldInitializers.join('\n        ')}
     }`)}
-
+    ${this.opts.wasm ? `` : `/** 
+     * Use this method to create a new, distinctly identified copy of the entity. 
+     * Storing the copy will result in a new copy of the data being stored.
+     */`}
     fun copy(${ktUtils.joinWithIndents(this.fieldsForCopyDecl, 14, 2)}) = ${name}(${ktUtils.joinWithIndents(this.fieldsForCopy, 8+name.length, 2)})
-
+    ${this.opts.wasm ? `` : `/** 
+     * Use this method to create a new version of an existing entity.
+     * Storing the mutation will overwrite the existing entity in the set, if it exists.
+     */
+    fun mutate(${ktUtils.joinWithIndents(this.fieldsForCopyDecl, 14, 2)}) = ${name}(${ktUtils.joinWithIndents(fieldsForMutate, 8+name.length, 2)})`}
```

Comment:
This is all a bit bunched up; I was attempting to avoid any changes (including newlines) in the generated wasm files